configUtil.js

```diff
@@ -26,8 +26,6 @@ module.exports = {
         'base-uri ': "'self'",
         'frame-ancestors': " 'none'",
         'form-action': "'none'",
-        'report-uri': 'https://csp-reporter.ente.io',
```

Comment:
This doesn't seem specific to local?

Reply:
these configurations are only for the local development server.
we were sending local reports to the same CSP-reporter webhook.
disabling that. as they are just adding noise to that group

Reply:
for production configs we have to _header file in /public folder

Reply:
@vishnukvmd